docs/examples/quantile_regression.ipynb

```diff
@@ -177,7 +178,7 @@
        "│ str              ┆ f64       ┆ u32        ┆ f64          ┆ f64         ┆ f64      │\n",
        "╞══════════════════╪═══════════╪════════════╪══════════════╪═════════════╪══════════╡\n",
        "│ linear_quant_reg ┆ 0.005     ┆ 200        ┆ 200.0        ┆ 0.030488    ┆ 0.869899 │\n",
-       "│ hgbt_quant_reg   ┆ 0.0       ┆ 200        ┆ 200.0        ┆ 0.030695    ┆ 1.0      │\n",
+       "│ hgbt_quant_reg   ┆ -0.025    ┆ 200        ┆ 200.0        ┆ 0.031653    ┆ 0.43057  │\n",
```

Comment:
I'm suprised to see a model change here. Maybe we should fix the `random_state` parameter of HGBT to some integer.

Reply:
As written in the development doc, you can run `hatch run jupyter:lab`. This will create a virtual environment with all you need to run the notebooks, and run jupyter lab.